README.md

```diff
@@ -228,6 +228,18 @@ For example, the following command adds a custom assertion called "org.contentau
 c2patool sample/image.jpg -c '{"assertions": [{"label": "org.contentauth.test", "data": {"my_key": "whatever I want"}}]}'
 ```
 
+### Speeding up signing
+
+By default c2patool will validate the signature immediately after signing a manifest.  This can be disabled to gain a slight speed up by using the  `--no_signing_verify` option.  
+
+### Configuring trust support
+
+Trust support is enabled by use the `trust` command and setting any combination of `trust` options. Use `trust_anchors` to specify a list of trust anchors (in PEM format) to use. The trust anchors are used to validate the manifest certificate chain. The manifest certificate chain must chain up to a certificate on the trust list to be considered valid. All certificates in the trust anchor list must have Basic Constraints extension CA: True. See sample `sample/trust_anchors.pem`. Use `allowed_list` to specify a list of end-entity certs (in PEM format) you wish to trust. These are the certificates used to sign the manifest. This check supersedes the `trust_anchors` check. The allowed list must not contain certificates with Basic Constraints extension CA: True.  See sample `sample/allowed_list.pem`. Use `trust_config` to specify a set of custom certificate extended key usages (EKUs) to allow. The format of configuration is a list object identifiers in Oid dot notation format. See sample `sample/store.cfg`.
```

Comment:
Here is my suggested reformatting.  I did not change the content/meaning:

Enable trust support by using the `trust` sub-command, as follows:

```
c2patool [path] trust [OPTIONS]
```

The following additional options are available with the `trust` sub-command:

| Trust option | Description | Example |
|--------------|-------------|---------|
| `--trust_anchors` | Specifies a list of trust anchors (in PEM format) used to validate the manifest certificate chain. The manifest certificate chain must chain up to a certificate on the trust list to be considered valid. All certificates in the trust anchor list must have `Basic Constraints extension CA: True`. | `sample/trust_anchors.pem` |
| `--allowed_list` | Supersedes the `trust_anchors` check and specifies a list of end-entity certificates (in PEM format) to trust. These certificates are used to sign the manifest. The allowed list must NOT contain certificates with `Basic Constraints extension CA: True`. |  `sample/allowed_list.pem`. |
| `--trust_config` | Specifies a set of custom certificate extended key usages (EKUs) to allow. Format is a list with object identifiers in OID dot notation format. | `sample/store.cfg`. |

Reply:
Can we provide more info than "OID dot notation format." e.g. a link.  Googling didn't bring up much but I presume it's relatively straightforward.

Reply:
@crandmck  Feel free to edit the PR.  This is your area of expertise.  For Oid it is just a registered number that represents something.  In this case it is for EKUs.  If you know what an EKU and need to add one you would know its Oid.  http://www.oid-info.com/#oid